cpp/src/neighbors/ann_mg/ann_mg_cagra_int8_t_uint32_t.cu

```diff
@@ -0,0 +1,93 @@
+/*
+ * Copyright (c) 2024, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/*
+ * NOTE: this file is generated by generate_ann_mg.py
+ *
+ * Make changes there and run in this directory:
+ *
+ * > python generate_ann_mg.py
+ *
+ */
+
+#include "ann_mg.cuh"
+
+namespace cuvs::neighbors::mg {
+
+#define CUVS_INST_ANN_MG_CAGRA(T, IdxT)                                                          \
```

Comment:
Can you please verify how much your changes increase the binary size and compile times compared to cuvs in branch-24.10? You changes aren't creating any new crazy complex kernels so ideally they shouldn't drastically increase binary size or compile time. However it's worth checking so we're not surprised once we merge it over.

Reply:
The PR adds 7M, from 130M to 137M.